packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -307,7 +311,12 @@ const Updates = decorate([
                 </p>
                 {state.showPackagesList && (
                   <p>
-                    <Copiable tagName='pre'>{state.packagesList}</Copiable>
+                    <Copiable tagName='pre'>
+                      <p>
+                        {_('xoaBuild')} {state.xoaBuild}
+                      </p>
+                      <span>{state.packagesList}</span>
```

Comment:
`packagesList` is a block element.

Reply:
Why? 
I need to display it in an `inline-block` because it's a `Copiable` children.

Reply:
we don't want to package list be inline with an other element.